BookAPI/Program.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 
 namespace BookAPI
 {
     public class Program
     {
+        static BookRepository bookRepository = new BookRepository();
         public static List<BookModel> bookList = new List<BookModel>();
```

Comment:
doesnt need bookList anymore?

Reply:
Ideally BookRepository must expose only AddBook method, all the validations must be private methods.